ooniprobe/View/TestResults/Footer/TestDetailsFooterViewController.m

```diff
@@ -31,4 +37,10 @@ - (void)viewDidLoad {
     [self.runtimeDetailLabel setTextColor:[UIColor colorWithRGBHexString:color_gray9 alpha:1.0f]];
 }
 
+-(IBAction)openMethodology:(id)sender{
```

Comment:
@lorenzoPrimi is it correct to say that this `openMethodology` part is unrelated with uploads and instead is the UI feature that allows to open the methodology of a specific nettest?

Reply:
Exactly. Bus as the storyboard was touched heavily in both branches I had to merge the methodology branch here. I couldn't be merged in master.

Reply:
So, it seems the bottleneck is really the storyboard. I'm starting to wonder whether there are smart solutions around always hitting the storyboard and or best practices. _/me reads stuff_

Reply:
So, [there is people suggesting to break the storyboard so that each view controller has its own](https://medium.com/practical-ios-development/how-real-teams-deal-with-ios-storyboards-beb8bb4c2765).

Reply:
[Here they compare the pros and cons of storyboard and XIB](https://stackoverflow.com/questions/9404471/when-to-use-storyboard-and-when-to-use-xibs).

Reply:
My point is the following: we should ask ourself the question of whether, if when we try to do sane coding the storyboard gets in the way, maybe it's not our fault, maybe the storyboard is the problem.

Reply:
Xib is so 2013 ! 
I used the approach to group storyboard in categories, like results.storyboard, settings.storyboard etc.. in this case is really difficult to have conflicts unless you really modify the same group of screens. 
Usually it shouldn't happen in fact I tend to add only one feature per version, in this case the methodology was heavily pushed and I should have waited this pr to be done to develop methodology

Reply:
> I used the approach to group storyboard in categories
Right, this seems a good way to handle the problem.

> [...] I should have waited this pr to be done to develop methodology
I guess we can work on as many non UI features as we want, but only one UI feature at a time per storyboard can be in development at any single time. Does this make sense?

Reply:
Exactly. Also I plan to divide the Result storyboard into two in the future. My approach is to keep max 7-8 screens inside a storyboad. Unless they are striclty connected for preview reasons.

Reply:
Ok, sounds reasonable!